### ISSUE TITLE
[Part 2 (9.2.0 Downgrade)] 11.0.0-33E after DSiware downgrade - formatting?

### ISSUE THREAD

**Issue by etianhelvoid** [Plailect/Guide#252]
Hello, I've got a hacked 2DS and I've done the DSiware downgrade on my 3DS XL (as far as I know successfully as I can even enter the homebrew using oot3d for the downgraded version which was the same I used on the 2DS which unlike the XL wasn't on 11.0.0). Since I softbricked an ambassador 3DS because I missed the warning to format it I have a question, do I need to format my 3DS XL? (it has the DSiware of Legends of Exidia with the hacked save) Does it affect the DSiware downgrade? Do I lose it if I do? Do I revert to 11.0.0-33?
## 
**Long story short may I format the 3DS XL or does it undo the DSiware downgrade?**

The reason I'm so wary is because I can't mess this up or I won't be able to try and hack it anytime soon.

Thanks in advance. :)

**Comment by Plailect** [comment]
You can format, it won't mess anything up.

**Comment by etianhelvoid** [comment]
Thank you very much!
Great guide ;)
